### PR TITLE
[LICENSE] Add MIT license to the repository

### PR DESCRIPTION
**Pull request by dz333n**
...and include correct authors

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+TODO: Actual Authors Here
```
Comment (learn-more) [inline LICENSE:3]: This does not look like `...and include correct authors`

Comment (dz333n) [reply LICENSE:3]: Agree! I'm not sure that this project is licensed under MIT license, so let's wait for actual authors answers!

**Review by prozacchiwawa — COMMENTED** [review]
That's fine with me.

**Comment by gedmurphy** [comment]
> That's fine with me.

Oh hey Art :)

**Comment by prozacchiwawa** [comment]
:-)

**Review by learn-more — CHANGES_REQUESTED** [review]
Please get the authors of this to sign off on it,
and include their names.

**Pull request closed without merging**